lib/Fixer.ts

```diff
@@ -55,6 +56,10 @@ export abstract class Fixer {
     if (opts.symbols) {
       filteredOpts.symbols = opts.symbols;
     }
+    
```

Comment:
Still tailing whitespace at the beginning of the line here
> ERROR: lib/Fixer.ts[59, 1]: trailing whitespace

Your can check it by running `npm run tslint`.